shell/edit/provisioning.cattle.io.cluster/DrainOptions.vue

```diff
@@ -10,7 +10,6 @@ const DEFAULTS = {
   force:                           false, // Show; true = Delete standalone pods, false = fail if there are any
   gracePeriod:                     -1, // Show; Pod shut down time, negative value uses pod default
   ignoreDaemonSets:                true, // Hide; true = work, false = never work because there's always daemonSets
-  ignoreErrors:                    false, // Hide; profit?
```

Comment:
What does removing this do?

Reply:
This field is also left out of the API response on creation. Removing this and `hostnamePrefix`'s ensures that when a user edits a cluster the UI doesn't add anything behind-the-scenes, because that causes the cluster to go into updating